### PR TITLE
The codes of BWT and FWT

### PR DESCRIPTION
**Pull request by hsj576**
**What type of PR is this?**

/kind feature

**What this PR does / why we need it**:

This PR provide codes for system metrics of lifelong learning in Ianvs.

Fixes #

### CODE REVIEW

File: examples/robot/lifelong_learning_bench/README.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+cd /ianvs/project/ianvs
+ianvs -f examples/robot/lifelong_learning_bench/benchmarkingjob.yaml
+```
+
+Finally, the user can check the result of benchmarking on the console and also in the output path(
+e.g. `/ianvs/lifelong_learning_bench/workspace`) defined in the benchmarking config file (
+e.g. `benchmarkingjob.yaml`). In this quick start, we have done all configurations for you and the interested readers
+can refer to [benchmarkingJob.yaml](https://ianvs.readthedocs.io/en/latest/guides/how-to-test-algorithms.html#step-1-test-environment-preparation) for more details.
+
+The final output might look like this:
```
Comment (luosiqi) [inline examples/robot/lifelong_learning_bench/README.md:93]: Is this the real output of this example? Could you show us some systematic performance, such as latency of the whole pipeline?

Comment (hsj576) [reply examples/robot/lifelong_learning_bench/README.md:93]: > Is this the real output of this example? Could you show us some systematic performance, such as latency of the whole pipeline?

Yes, it is the real output of this example. The tutorial of this example could be found in [here](https://github.com/hsj576/ianvs-lifelong-learning-tutorial/blob/main/tutorial/tutorial.md).
result

File: core/__init__.py
Comment (jaypume) [inline core/__init__.py:1]: Why so many file's permission have been changed? Is this change necessary?

Comment (hsj576) [reply core/__init__.py:1]: > Why so many file's permission have been changed? Is this change necessary?
The change is not necessary.  It's just an accident, I will recover it.

Comment (hsj576) [reply core/__init__.py:1]: > Why so many file's permission have been changed? Is this change necessary?

The change is an accident, I will recover it.

**Comment by MooreZheng** [comment]
Have heavily discussed in routine meetings with all comments revised.
